x-pack/plugins/apm/public/components/app/service_overview/index.tsx

```diff
@@ -98,30 +97,7 @@ export function ServiceOverview({
               </EuiFlexItem>
               <EuiFlexItem grow={6}>
                 <EuiPanel>
-                  <EuiFlexGroup>
-                    <EuiFlexItem>
-                      <EuiTitle size="xs">
-                        <h2>
-                          {i18n.translate(
-                            'xpack.apm.serviceOverview.dependenciesTableTitle',
-                            {
-                              defaultMessage: 'Dependencies',
-                            }
-                          )}
-                        </h2>
-                      </EuiTitle>
-                    </EuiFlexItem>
-                    <TableLinkFlexItem>
-                      <ServiceMapLink serviceName={serviceName}>
-                        {i18n.translate(
-                          'xpack.apm.serviceOverview.dependenciesTableLinkText',
-                          {
-                            defaultMessage: 'View service map',
-                          }
-                        )}
-                      </ServiceMapLink>
-                    </TableLinkFlexItem>
-                  </EuiFlexGroup>
+                  <ServiceOverviewDependenciesTable serviceName={serviceName} />
```

Comment:
I think we could drop the `Overview` in the name, `ServiceDependenciesTable` is clear enough and can be reused elsewhere.

```suggestion
                  <ServiceDependenciesTable serviceName={serviceName} />
```